src/lwc/geFormField/__mocks__/geFormField.js

```diff
@@ -0,0 +1,7 @@
+import { LightningElement, api } from 'lwc';
```

Comment:
@bdvorachek - added a local mock and would love to get your early thoughts here.

Reply:
I don't think we'd need to mock geFormField here, this folder would be for mocks that are specific to this component's tests. We could move the mock implementation of geFormService from geFormService.test.js into this folder as its own file and use it in the test.